scripts/Slngen.ps1

```diff
@@ -70,7 +70,7 @@ param (
     [Parameter(Mandatory = $false, HelpMessage="Minimizes console output.")]
     [switch]$Quiet = $false,
     [Parameter(Mandatory = $false, HelpMessage="Output file name.")]
-    [string]$OutSln = "SDK.sln",
+    [string]$OutSln = "Extensions.sln",
```

Comment:
This change will lead to chaos
![image](https://github.com/dotnet/extensions/assets/4403806/e720280c-3f4a-423e-81f0-26ca00d53d9d)

Reply:
Undone.